### DISCUSSION TITLE
Lower case "report"

### DISCUSSION THREAD

**Discussion by robeya123** [ThioJoe/YT-Spammer-Purge#247] [Ideas]
Hi,

Could you make it so the lower case also works? I typed in "report" instead of "REPORT" and the operation was cancelled only giving me the option of quitting.

**Comment by ThioJoe** [comment]
I specifically wanted the final "actions" (report, hold, delete) to require a tiny bit of extra effort so people would be sure they want to. I might change it in the future but it is that way on purpose
